src/data/icons.json

```diff
@@ -696,6 +696,48 @@
       "name": "spraybottle",
       "desc": "Indicates that there is information about cleaning",
       "category": "health"
+    },
+    {
+      "title": "clock",
+      "name": "clock-filled",
+      "desc": "Clock face with hands",
+      "category": "interface"
+    },
+    {
+      "title": "clock",
+      "name": "clock-stroke",
+      "desc": "Clock face with hands",
+      "category": "interface"
+    },
```

Comment:
Unique descriptions for how these two are different would be helpful.

Reply:
Why? There is only a slight visual difference between them. The description is mainly used as an accessibility tool. Is there real value in describing of the clock is filled or outlined?